docker/images/alluxio/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:8
```

Comment:
i'm ignoring all the docker stuff in this review since this should be cleaned up in the other branch.

Reply:
(y)